src/main/java/eu/erasmuswithoutpaper/registryclient/CatalogueDocument.java

```diff
@@ -305,14 +353,20 @@ public Iterator<String> getPrefixes(String namespaceUri) {
 
         mapForType.put(getCanonicalId(value), heiId);
       }
-      for (Element heiElem : Utils.asElementList(
```

Comment:
Why did you extract variables from the following "for" loops?

Reply:
It looks strange but the debugger was evaluating the full expression for each loop

Reply:
So this is only done for debugging purposes and doesn't speed up production code?

I wanted to suggest reverting those changes, as they make the code "less clean". What do you think?

Reply:
It is not only for debugging but also to have it cleaner.
The less you do in a line of code, the cleaner it gets.

Reply:
It seems to me that it's better to not define variables that are not used outside the loop but maybe let's not dwell on that. I can accept the current change after I make sufficient tests.

Thanks for you help!